### PR TITLE
Virtual templates

### PR DESCRIPTION
**Pull request by jntullo**
Adds virtual templates to the docs

### CODE REVIEW

File: api/reference/virtual_templates.adoc [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+* link:#querying-virtual-templates[Querying Virtual Templates]
+* link:#creating-virtual-templates[Creating Virtual Templates]
+* link:#editing-virtual-templates[Editing Virtual Templates]
+* link:#deleting-virtual-templates[Deleting Virtual Templates]
+
+[[querying-virtual-templates]]
+=== Querying Virtual Templates
+
+Querying all virtual templates in the system is simply:
+
```
Comment (mfeifer) [inline api/reference/virtual_templates.adoc]: remove "simply". you don't need it. Either use "Getting" or "Querying" to be consistent.

File: api/reference/virtual_templates.adoc [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+{
+    "name": "virtual_template1",
+    "vendor": "amazon",
+    "cloud_subnet_id": "300",
+    "availability_zone_id": "245",
+    "type":"ManageIQ::Providers::Amazon::CloudManager::VirtualTemplate"
+}
+----
+
+or creating multiple virtual templates:
```
Comment (mfeifer) [inline api/reference/virtual_templates.adoc]: remove "or"

File: api/reference/virtual_templates.adoc [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      "cloud_subnet_id": "345",
+      "availability_zone_id": "562",
+      "type":"ManageIQ::Providers::Google::CloudManager::VirtualTemplate"
+    }
+  ]
+}
+----
+
+[[editing-virtual-templates]]
+=== Editing Virtual Templates
```
Comment (mfeifer) [inline api/reference/virtual_templates.adoc]: Editing a Virtual Template (singular)

File: api/reference/virtual_templates.adoc [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+----
+{
+  "action": "edit",
+  "resource": {
+    "name": "edited_name"
+  }
+}
+----
+
+or editing multiple virtual templates:
```
Comment (mfeifer) [inline api/reference/virtual_templates.adoc]: remove "or"

File: api/reference/virtual_templates.adoc [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+----
+
+[source,json]
+----
+{
+  "action": "delete"
+}
+----
+
+or simply:
```
Comment (mfeifer) [inline api/reference/virtual_templates.adoc]: remove "simply"

**Comment by mfeifer** [comment]
In addition, to my inline comments.  For consistency, you should either have the commands for multiple/all templates come first or second in each section.

**Comment by Fryguy** [comment]
@jntullo This will probably need to be rewritten in the new directory now that #62 has been merged.  Note that the current REST API documentation is still in the repo, but it will eventually be moved/merged into the new directory. cc @abellotti @imtayadeway

**Comment by jntullo** [comment]
@Fryguy will do. Was actually waiting to update this PR just because of some other changes we're working on. I'll actually close this for the time being.

**Pull request closed without merging**